custom_components/econnect_metronet/devices.py

```diff
@@ -154,6 +155,11 @@ def update(self):
             _LOGGER.error(f"Device | Error while checking if there are updates: {err}")
             raise
 
+        # Update the _inventory
+        self._inventory.update({"inputs": inputs["inputs"]})
+        self._inventory.update({"sectors": sectors["sectors"]})
+        self._inventory.update({"alerts": alerts})
```

Comment:
This is a note for later (probably you'll reach this challenge in one of the following PRs), but I think we should change the code in `econnect-python` so that instead of having:
```python
self._connection.get_status()
```

we have:
```python
self._connection.query(q.ALERTS)  # or whatever q.<NAME>
```

I think it's better because we create a single API (`.query()`) that developers know they should use when they want to query the system. Also, `query(q.ALERTS)` should return a dictionary exactly as `inputs` and `sectors` so that the result of the call is something expected.

This is a breaking change we should make in `econnect-python`, but we can delay the change in one of the following PRs. I would suggest to keep a note of this comment when you work on the refactoring.